docs/usage/updating.md

```diff
@@ -25,6 +25,27 @@ The `racker upgrade` on the other hand, will pull whatever the latest tag for Ra
 **Important:** This means that running `racker upgrade` can bring new versions of tools/modules (`lokoctl`, `terraform`, etc.) that may be incompatible with the ones currently deployed (which means that e.g. the Lokomotive configuration may have to be manually updated to fit the new version of `lokoctl`).
 For this reason, upgrading Racker should be done only when no cluster is yet deployed, or when the cluster has been wiped out and the intent is to start over with the very latest Racker version.
 
+## Notes for upgrading to Racker 0.3
+
+The jump from Racker 0.2 to 0.3 through `racker upgrade` requires to add the following entry to the `baremetal.lokocfg` file to keep the behavior of reprovisioning worker nodes on configuration changes:
+
+```
+ignore_worker_changes = false
+```
+
+The Racker 0.3 release is using Lokomotive v0.9.0 with no modifications as all Racker changes are now upstreamed.
+The [release notes of Lokomotive v0.9.0](https://github.com/kinvolk/lokomotive/releases/tag/v0.9.0) are valid except for the update steps for the Baremetal platform which are reduced to the change mentioned above.
+
+After changing the `baremetal.lokocfg` file as metioned, run the following steps:
+
+```
+racker upgrade
+lokoctl cluster apply --skip-components
+curl -LO https://github.com/kinvolk/lokomotive/archive/v0.9.0.tar.gz
+tar -xvzf v0.9.0.tar.gz
+lokoctl components apply
```

Comment:
Need to mention that user needs to run the update script to update the hosted kubelet and etcd versions.

Reply:
Good catch

Reply:
done